projection/validator/validator.go

```diff
@@ -511,3 +603,83 @@ func (projection *Validator) projectValidatorView(
 
 	return nil
 }
+
+func (projection *Validator) checkAttentionOnCommission(mutValidatorRow *view.ValidatorRow, maybeCommissionRate string, commissionRate string, validatorAddress string) error {
+	if projection.config.AttentionStatusRules.MaxCommissionRateChange.Enable {
+		newCommission, newCommissionErr := strconv.ParseFloat(maybeCommissionRate, 64)
+		if newCommissionErr != nil {
+			return fmt.Errorf(
+				"error converting new commission rate to float validator %s from view", validatorAddress,
+			)
+		}
+		currentCommission, currentCommissionErr := strconv.ParseFloat(commissionRate, 64)
+		if currentCommissionErr != nil {
+			return fmt.Errorf(
+				"error converting current commission rate to float validator %s from view", validatorAddress,
+			)
+		}
+
+		if newCommission-currentCommission > projection.config.AttentionStatusRules.MaxCommissionRateChange.MaxChange {
+			mutValidatorRow.Status = constants.ATTENTION
+		}
+	}
+
+	return nil
+}
+
+func (projection *Validator) checkAttentionOnNumOfChanges(mutValidatorRow *view.ValidatorRow, mutValidatorActivities []view.ValidatorActivityRow, editField string) {
```

Comment:
suggest to change`checkAttentionOnNumOfChanges ` > `isExcessNumOfEdit` while return a boolean